sdcflows/interfaces/bspline.py

```diff
@@ -171,26 +176,25 @@ def _run_interface(self, runtime):
             else np.asanyarray(masknii.dataobj) > 1e-4
         )
 
-        # Convert spacings to numpy arrays
-        bs_spacing = [np.array(sp, dtype="float32") for sp in self.inputs.bs_spacing]
-
         # Recenter the fieldmap
         if self.inputs.recenter == "mode":
             from scipy.stats import mode
 
-            data -= mode(data[mask], axis=None)[0][0]
+            # Handle pre- and post-1.9 mode behavior.
+            # squeeze can be dropped when the minimum version reaches 1.9
+            data -= np.squeeze(mode(data[mask]).mode)
```

Comment:
Looks like we set the min scipy to 1.9.2 in #304.

```suggestion
            data -= mode(data[mask], keepdims=False).mode
```

Reply:
Moved out to #309 - I forgot to type the `Co-authored-by` metadata, do you want me to force-push or send your own PR?

Reply:
Squashed-and-merged and added it to the squash commit.